src/transformers/visitors/toCircuitVisitor.ts

```diff
@@ -108,7 +108,7 @@ const visitor = {
       if (state.msgSenderParam) {
         node._newASTPointer.parameters.parameters.unshift(
           buildNode('VariableDeclaration', {
-            name: 'msgSender',
+            name: 'msg',
```

Comment:
As above, the address in the circuit corresponding to `msg.sender` is now named `msg` everywhere